src/Aspire.Cli/ExitCodeConstants.cs

```diff
@@ -15,4 +15,5 @@ internal static class ExitCodeConstants
     public const int FailedToFindProject = 7;
     public const int FailedToTrustCertificates = 8;
     public const int AppHostIncompatible = 9;
+    public const int InputCanceled = 10;
```

Comment:
I think if we just throw an OperationCancelledException in most cases the appropriate error code will be returned. No need to introduce a new one.

Reply:
I'll remove the exception and this constant